front/app/api/idea_statuses/types.ts

```diff
@@ -18,40 +17,52 @@ export type IdeaStatusesQueryParams = {
   participation_method?: IdeaStatusParticipationMethod;
 };
 
-type InputStatusCode =
+type IdeationStatusCode =
+  | 'prescreening'
   | 'proposed'
   | 'viewed'
   | 'under_consideration'
   | 'accepted'
   | 'rejected'
   | 'implemented'
-  | 'custom'
+  | 'custom';
+type ProposalsStatusCode =
+  | 'prescreening'
+  | 'proposed'
   | 'threshold_reached'
   | 'expired'
   | 'answered'
-  | 'ineligible';
+  | 'ineligible'
+  | 'custom';
+export type InputStatusCode = IdeationStatusCode | ProposalsStatusCode;
+
+const ideationInputStatusCodes: IdeationStatusCode[] = [
+  'prescreening',
+  'proposed',
+  'viewed',
+  'under_consideration',
+  'accepted',
+  'rejected',
+  'implemented',
+  'custom',
+];
+
+const proposalsInputStatusCodes: ProposalsStatusCode[] = [
+  'prescreening',
+  'proposed',
+  'threshold_reached',
+  'expired',
+  'answered',
+  'ineligible',
+  'custom',
+];
```

Comment:
Not the best type solution, but a slight improvement over the previous version. I can't add a proposals code to the ideation codes array without getting an error, which I could before.

Reply:
It seems like a good solution to me 👍